modules/installation-special-config-storage.adoc

```diff
@@ -11,37 +11,51 @@ During an {product-title} installation, you can enable boot disk encryption and
 [id="installation-special-config-encrypt-disk_{context}"]
 == About disk encryption
 
-You can enable encryption for the boot disks on the control plane and compute nodes at installation time. {product-title} supports the Trusted Platform Module (TPM) v2 and Tang encryption modes.
+You can enable encryption for the boot disks on the control plane and compute nodes at installation time.
+{product-title} supports the Trusted Platform Module (TPM) v2 and Tang encryption modes.
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor contained within a server. You can use this mode to prevent the boot disk data on a cluster node from being decrypted if the disk is removed from the server.
-* Tang: Tang and Clevis are server and client components that enable network-bound disk encryption (NBDE). You can bind the boot disk data on your cluster nodes to one or more Tang servers. This prevents the data from being decrypted unless the nodes are on a secure network where the Tang servers can be accessed. Clevis is an automated decryption framework that is used to implement the decryption on the client side.
+* TPM v2: This is the preferred mode.
```

Comment:
Nit: Perhaps use definition lists for these items? https://docs.asciidoctor.org/asciidoc/latest/lists/description/ 

```suggestion
TPM v2:: This is the preferred mode.
```